css/style.css

```diff
@@ -0,0 +1,281 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;  
+}
+
+@font-face {
+    font-family: 'myFont';
+    src: url(myFont.otf);
+}
+
+.menu {
+    width: 100%;
+    height: 100vh;
+    position: relative;
+}
+
+nav {
+    width: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    padding: 20px 8%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between; 
+    font-family: 'Blanka', sans-serif;
+}
+
+nav ul li{
+    list-style: none;
+    display: inline-block;
+    margin-left: 40px;
+}
+
+nav ul li a{
+    color: antiquewhite;
+    text-decoration: none;
+    font-size: 20px;
+}
+
+.home-video{
+     position: absolute;
+     right: 0;
+     bottom: 0;
+     z-index: -1;
+}
+
+@media (min-aspect-ratio: 16/9){
+    .home-video{
+        width: 100%;
+        height: auto;
+    }
+}
+
+@media (max-aspect-ratio: 16/90){
+    .home-video{
+        width: auto;
+        height: 100%;
+    }
+}
+
+h1 {
+    font-family: 'Blanka', sans-serif;
+    color: antiquewhite;
+    text-align: center;                                           
+    font-size: 75px;
+    font-weight: bold;
+    margin: 150px;
+    padding: 30px;
+}
+
+body{
+    background-color: black;
+  }
+
+#about {
+    text-align: center;
+    color: white;
+    font-size: 20px;
+    margin-top: 50px;
+    padding-top: 50px;
+    padding-bottom: 100px;
+}
+
+p{ 
+    border-style: none;
+    text-align: justify;
+    margin-top: 100px;
+    margin-bottom: 100px;
+    margin-right: 200px;
+    margin-left: 140px;
+    font-size: 20px;
+    font-family: monaco,Consolas,Lucida Console,monospace; 
+    height: 200px;
+    width: 50%;
+}
+
+img {
+    float:right;
+    width: 70px;
+    height:60px;
+}
+
+#img-riri {
+    float:right;
+    width: 300px;
+    height:300px;
+    margin-right: 180px;  
+}
+
+#experience {
+    text-align: center;
+    font-size: 20px;
+    border-style: hidden;
+    padding-top: 100px;
+    padding-bottom: 100px;
+}    
+
+table, th, td {
+    border: 1px solid black;
+    align-content: center;
+    position: relative;
+    margin-left: 250px;
```

Comment:
On smaller screens this margin makes the table get kinda pushed off the scren. You could use a media query to handle this or instead of using a hardcoded pixel value for the margin you could experiment with percentage units or `vw` ([viewport width](https://www.sitepoint.com/css-viewport-units-quick-start/)) units 

![](https://user-images.githubusercontent.com/9094098/207615990-bb4522e7-dc16-40f6-8a3d-daa558517ba8.png)